VAMobile/src/store/api/demo/mocks/prescriptions.json

```diff
@@ -542,7 +542,7 @@
           "quantity": null,
           "refillDate": null,
           "refillRemaining": null,
-          "refillStatus": "suspended",
+          "refillStatus": "active",
```

Comment:
May want to try running the relevant E2E tests (Prescriptions.e2e.ts) to make sure the changes to demo mode data don't break the tests

Reply:
good call. I removed the e2e tests that referenced suspended and non-verified. It was just the filter drop down